### ISSUE TITLE
error validating data: [found invalid field initialDelaySeconds for v1.HTTPGetAction, found invalid field timeoutSeconds for v1.HTTPGetAction]

### ISSUE THREAD

**Issue by SuperNami** [kubernetes/helm#1775]
After upgrading Tiller and the CLI to the latest version (v2.1.3) I got the following error while running my chart:

`Error: release unrealized-rattlesnake failed: error validating "": error validating data: [found invalid field initialDelaySeconds for v1.HTTPGetAction, found invalid field timeoutSeconds for v1.HTTPGetAction, found invalid field initialDelaySeconds for v1.HTTPGetAction, found invalid field timeoutSeconds for v1.HTTPGetAction, found invalid field limits for v1.Probe, found invalid field requests for v1.Probe, found invalid field resources for v1.Probe]`

It's pointing to the health check features:

```
        livenessProbe:
          httpGet:
            path: /
            port: http
            initialDelaySeconds: 120
            timeoutSeconds: 5
        readinessProbe:
          httpGet:
            path: /
            port: http
            initialDelaySeconds: 5
            timeoutSeconds: 1
```

These settings are exactly the same as official helm charts, so it shouldn't be failing.

Chart is located here: https://github.com/SuperNami/helm-charts

**Comment by technosophos** [comment]
Which version of Kubernetes?

**Comment by SuperNami** [comment]
[user@pc ~]$ kubectl version
Client Version: version.Info{Major:"1", Minor:"4", GitVersion:"v1.4.6", GitCommit:"e569a27d02001e343cb68086bc06d47804f62af6", GitTreeState:"clean", BuildDate:"2016-11-12T05:22:15Z", GoVersion:"go1.6.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"4", GitVersion:"v1.4.7", GitCommit:"92b4f971662de9d8770f8dcd2ee01ec226a6f6c0", GitTreeState:"clean", BuildDate:"2016-12-10T04:43:42Z", GoVersion:"go1.6.3", Compiler:"gc", Platform:"linux/amd64"}

**Comment by technosophos** [comment]
Oh! The issue is actually in the YAML: `initialDelaySeconds is not a property of httpGet, it's a property of livenessProbe or readinessProbe: http://kubernetes.io/docs/user-guide/liveness/

**Comment by SuperNami** [comment]
Thanks, I missed it. Problem solved.

**Comment by sb1975** [comment]
I had similar issue  :
$ helm install --name grafana stable/grafana
Error: error validating "": error validating data: [found invalid field timeoutSeconds for v1.HTTPGetAction, found invalid field failureThreshold for v1.HTTPGetAction, found invalid field initialDelaySeconds for v1.HTTPGetAction, found invalid field periodSeconds for v1.HTTPGetAction]

$ kubectl version
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.1", GitCommit:"d4ab47518836c750f9949b9e0d387f20fb92260b", GitTreeState:"clean", BuildDate:"2018-04-12T14:26:04Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.6", GitCommit:"9f8ebd171479bec0ada837d7ee641dec2f8c6dd1", GitTreeState:"clean", BuildDate:"2018-03-21T15:13:31Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}

so I did the following to resolve this :
$ helm fetch stable/grafana
$ tar -zxvf grafana-1.0.2.tgz

$ grep -r periodSeconds grafana/
grafana/templates/deployment.yaml:              periodSeconds: 10
$ vi grafana/templates/deployment.yaml
edit to ensure
periodSeconds ,initialDelaySeconds,etc all are in line with the httpget
          readinessProbe:
            httpGet:
              path: /api/health
              port: 3000
            initialDelaySeconds: 60
            timeoutSeconds: 30
            failureThreshold: 10
            periodSeconds: 10

helm install ./grafana --name grafana
done!!

**Comment by dimitrovvlado** [comment]
Fixed with chart version 1.2.0
